WGLMakie/src/serialization.jl

```diff
@@ -152,7 +152,11 @@ function flatten_buffer(array::AbstractArray{T}) where {T}
     return flatten_buffer(collect(reinterpret(eltype(T), array)))
 end
 
-lasset(paths...) = read(joinpath(@__DIR__, "..", "assets", paths...), String)
+const ASSETS_DIR = @path joinpath(@__DIR__, "..", "assets")
+function lasset(paths...)
+    asset_path = @path joinpath(ASSETS_DIR, paths...)
```

Comment:
hmm.  i see your point about my use of `@path` being in a function, but the [RelocatableFolders.jl docs](https://github.com/JuliaPackaging/RelocatableFolders.jl?tab=readme-ov-file#limitations) are pretty clear that if you `joinpath` on ASSETS_DIR you need to invoke `@path` again.  perhaps `lasset` needs to be refactored out of the code entirely.  cc @jkrumbiegel

Reply:
If I understand it correctly, `RelocatableFolders` serializes the complete directory with all subdirectories and their files. So this part of the README

```julia
using RelocatableFolders

const ASSETS = @path joinpath(@__DIR__, "../assets")
const SUBDIR = @path joinpath(ASSETS, "subdir")
const FILE = @path joinpath(ASSETS, "file.txt")
```

would be confusing in the sense that `SUBDIR` and `FILE` are already relocatable just through `ASSETS`.

I think that's also where your problem with the fonts came from, normally if you `joinpath` on the output of `@path` you get out a string so that works fine.